src/scene/particle-system/particle-emitter.js

```diff
@@ -13,7 +13,7 @@ import {
     BUFFER_DYNAMIC,
     CULLFACE_NONE,
     FILTER_LINEAR, FILTER_NEAREST,
-    INDEXFORMAT_UINT16,
+    INDEXFORMAT_UINT32,
```

Comment:
Why did Uint16 used to be the limit?

Reply:
Because for WebGL1, uint32 index was an extension. For WebGL2, this is part of the API and we use it all the time.